.github/workflows/scan-from-published-image.yaml

```diff
@@ -0,0 +1,104 @@
+name: Scan
```

Comment:
This looks good. I 'm thinking if we should unify this and https://github.com/canonical/charmed-kubeflow-workflows/blob/main/.github/workflows/scan-rock.yaml. This would require some `if`s in order to make it work for both cases I guess. WDYT? Maybe keeping those separately can make for a cleaner end result

Reply:
I was planning on doing this, but then hit the limit of nested workflows that we can have, and then decided to abandon the idea.

Reply:
I don't think this should play a role here. I mean the amount of nesting should be the same given that we 'd replace this https://github.com/canonical/charmed-kubeflow-workflows/pull/73/files#diff-dd35b90c2446ee9fa26f8d74002a710fdda5ef2a01bc1e337971ce690d2e9551R15 with the updated `scan-rock.yaml` file and the new call would be 
```
  scan:
    uses: ./.github/workflows/scan-rock.yaml # we can always update this name
    secrets: inherit
    with:
      image-name: ${{ inputs.image-name }}
      severity: ${{ inputs.severity }}

Reply:
I was getting an actual error, but I cannot seem to find it from the runs. I can explore the option again, I also think it would be ideal to merge the scan workflows.

Reply:
Yes let's do it this way! When you create an issue about it, feel free to resolve this comment

Reply:
Issue: https://github.com/canonical/charmed-kubeflow-workflows/issues/77